app/helpers/application_helper.rb

```diff
@@ -6,4 +6,8 @@ def format_bidder(high_bidder)
       "None"
     end
   end
+
+  def order_items_by_id(items)
```

Comment:
I found that it was sorting by updated at (I think) and since we're displaying the item id for the admin i figured it would make most sense to just sort by id